crypto/stark/Cargo.toml

```diff
@@ -14,95 +14,68 @@ license = "Apache-2.0"
+zkp-logging-allocator = { version = "0.1.0", path = "../../utils/logging-allocator", optional = true }
+zkp-macros-decl = { version = "0.1.0", path = "../../utils/macros-decl", default-features = false }
+zkp-merkle-tree = { version = "0.1.0", path = "../../crypto/merkle-tree", default-features = false }
+zkp-mmap-vec = { version = "0.1.0", path = "../../utils/mmap-vec", default-features = false }
+zkp-primefield = { version = "0.1.1", path = "../../algebra/primefield", default-features = false }
+zkp-u256 = { version = "0.1.1", path = "../../algebra/u256", default-features = false }
 
-# Examples dependencies
-structopt = { version = "0.3.5", optional = true }
-rand = { version = "0.7.2", optional = true }
-rand_xoshiro = { version = "0.4.0", optional = true }
+[dev-dependencies]
+criterion = "0.3.0"
+env_logger = "0.6.2"
+proptest = "0.9.4"
+quickcheck = "0.9"
+quickcheck_macros = "0.8"
+rand = "0.7.2"
+rand_xoshiro = "0.4.0"
+structopt = "0.3.5"
+zkp-criterion-utils = { version = "0.1.0", path = "../../utils/criterion-utils" }
+zkp-elliptic-curve = { version = "0.1.0", path = "../../algebra/elliptic-curve" }
+zkp-elliptic-curve-crypto = { version = "0.1.0", path = "../../crypto/elliptic-curve-crypto" }
+zkp-logging-allocator = { version = "0.1.0", path = "../../utils/logging-allocator" }
+zkp-primefield = { version = "0.1.0", path = "../../algebra/primefield", features = ["quickcheck"] }
+zkp-u256 = { version = "0.1.0", path = "../../algebra/u256", features = ["quickcheck"] }
 
 [[bench]]
 name = "benchmark"
 harness = false
-required-features = ["bench"]
 
 [features]
-default = [ "std", "inline", "prover", "examples" ]
+default = [
+    "inline",
+    "prover",
+    "std",
+]
 std = [
-    "zkp-macros-decl/std",
-    "zkp-mmap-vec/std",
-    "zkp-u256/std",
-    "zkp-primefield/std",
-    "zkp-elliptic-curve/std",
-    "zkp-elliptic-curve-crypto/std",
-    "zkp-hash/std",
-    "zkp-merkle-tree/std",
-    "no-std-compat/std",
+    "hex",
     "itertools/use_std",
     "log/std",
+    "no-std-compat/std",
     "rayon",
-    "hex",
+    "zkp-hash/std",
+    "zkp-macros-decl/std",
+    "zkp-merkle-tree/std",
+    "zkp-mmap-vec/std",
+    "zkp-primefield/std",
+    "zkp-u256/std",
 ]
 inline = [
-    "zkp-u256/inline",
     "zkp-primefield/inline",
-    "zkp-elliptic-curve/inline",
-]
-test = [
-    "zkp-u256/test",
-    "zkp-primefield/test",
-    "zkp-elliptic-curve/test",
-    "zkp-elliptic-curve-crypto/test",
-    "proptest",
-    "quickcheck",
-    "quickcheck_macros",
-    "hex",
-    "env_logger",
-]
-bench = [
-    "criterion",
-    "zkp-criterion-utils",
-]
-examples = [
-    "zkp-primefield/use_rand",
-    "zkp-logging-allocator",
-    "zkp-elliptic-curve",
-    "zkp-elliptic-curve-crypto",
-    "env_logger",
-    "structopt",
-    "rand",
-    "rand_xoshiro",
+    "zkp-u256/inline",
 ]
 prover = [
     "std", # TODO: Make prove run in no-std wasm.
-    "zkp-merkle-tree/prover",
+    "zkp-merkle-tree/prover"
```

Comment:
did you mean to remove this comma? it seems inconsistent with the rest of the file.